client/packages/host/src/api/hooks/settings/useConfigureNameProperties.ts

```diff
@@ -48,14 +48,7 @@ export const useConfigureNameProperties = () => {
     const primaryConfig = propertyConfigurations[propertyType];
     const primaryProperties = localiseProperties(primaryConfig.properties);
 
-    const properties = [...primaryProperties];
-
-    Object.entries(propertyConfigurations).forEach(([type, config]) => {
-      if (type !== propertyType && config.isConfigured)
-        properties.push(...localiseProperties(config.properties));
-    });
-
-    return properties;
```

Comment:
I cannot figure out what the purpose of this extra code was for. Seems redundant to me, and I can't see any functional difference (it returns exactly the same thing, as far as I can tell)

Reply:
@GeronimoJohn -- apparently you were the last to edit this -- am I missing something silly here?